src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.js

```diff
@@ -176,19 +177,28 @@ var make_link = function(h, num_lbl) {
                 $('#notebook-container').css('margin-left', 30);
                 $('#notebook-container').css('width', $('#notebook').width() - 30);
             } else { // original width
-              $("#notebook-container").css({'width':"82%", 'margin-left':'auto'})             
+              $("#notebook-container").css({'width':'', 'margin-left':'auto'})             
```

Comment:
the margin-left should probably also be reset here (admittedly, it does end up as `auto` with the default css, but it's neater to just remove the override, in case anyone is trying to customize things)